### ISSUE TITLE
Pickled TnT tagger doesn't load cleanly

### ISSUE THREAD

**Issue by leondz** [nltk/nltk#293]
When pickling/cPickling a TnT tagger under protocol 0, 1 or 2, in this way:

``` python
# learn and save tagger

t0 = nltk.DefaultTagger('NN') # most frequent tag in train set

tnt_tagger = tnt.TnT(unk=t0, Trained=True, C=True)
tnt_tagger.train(train.tagged_sents())

print 'saving TnT tagger'
tagger_file = open(corpus_file + '.tnt-model', 'w')
pickle.dump(tnt_tagger, tagger_file, 1)
tagger_file.close()
```

 the load fails, with the message (from pickle.load()):

``` python
Traceback (most recent call last):
  File "./tag-nltk.py", line 16, in <module>
    tagger = pickle.load(tagger_file)
  File "/usr/lib/python2.7/pickle.py", line 1378, in load
    return Unpickler(file).load()
  File "/usr/lib/python2.7/pickle.py", line 858, in load
    dispatch[key](self)
  File "/usr/lib/python2.7/pickle.py", line 1133, in load_reduce
    value = func(*args)
  File "/usr/local/lib/python2.7/dist-packages/nltk-2.0.2-py2.7.egg/nltk/probability.py", line 1729, in __init__
    for (cond, sample) in cond_samples:
TypeError: 'type' object is not iterable
```

As a temporary fix, editing nltk/probability.py to ignore this exception leads to the unpickling yielding an apparently working tagger object;

``` python
       defaultdict.__init__(self, FreqDist)
        if cond_samples:
            try:
                for (cond, sample) in cond_samples:
                    self[cond].inc(sample)
            except:
                pass
```

**Comment by rmalouf** [comment]
This is the same problem as #393 and it's also fixed by #394, so this can probably by closed.

**Comment by alexrudnick** [comment]
This seems to be fixed for Python 2, but due to #398, can't verify it for Python 3 yet...

**Comment by alexrudnick** [comment]
OK, yeah! This can be closed. Thanks, Rob!
